kolibri/plugins/style_guide/assets/src/app.js

```diff
@@ -1,13 +1,16 @@
 const KolibriModule = require('kolibri_module');
 const Vue = require('kolibri.lib.vue');
 const RootVue = require('./views');
+const router = require('kolibri.coreVue.router');
```

Comment:
I did try using just "vue-router" actually, but it was conflicting with the other imports of "vue-router" elsewhere in the codebase. So I ended up having to use the Kolibri router wrapper instead.